calculator/src/ShuntingYard.kt

```diff
@@ -0,0 +1,52 @@
+
+
+fun shunt (inputList: List<Token>) : List<Token>
+{
+    var outputQueue= mutableListOf<Token>()
+    var operatorStack= mutableListOf<Token>()
+
+    for(x in inputList)
+    {
+        if(x is Floating || x is Integer)
+            outputQueue.enqueue(x)
+        else if(x is Function)
+            operatorStack.push(x)
+        else if(x is BinaryOp) {
```

Comment:
Inconsistent formatting?

PS Intellij has Ctrl-Atl-L to auto format

Reply:
formatted!